.github/pyinstaller/pyinstaller.spec

```diff
@@ -4,37 +4,19 @@ import sys
 import logging
 
 import wcwidth
+import capa.rules.cache
 
 from pathlib import Path
 
 logger = logging.getLogger(__name__)
 
-def generate_rule_cache(capa_dir: Path):
-    import capa
-    import capa.rules
-    import capa.rules.cache
-
-    rules_dir = capa_dir / 'rules'
-    cache_dir = capa_dir / 'cache'
-
-    try:
-        cache_dir.mkdir(parents=True, exist_ok=True)
-        rules = capa.rules.get_rules([Path(rules_dir)], cache_dir)
-        logger.info(f"successfully loaded {len(rules)} rules")
-    except (IOError, capa.rules.InvalidRule, capa.rules.InvalidRuleSet) as e:
-        logger.error(f"{str(e)}")
-        sys.exit(-1)
-
-    content = capa.rules.cache.get_ruleset_content(rules)
-    id = capa.rules.cache.compute_cache_identifier(content)
-    path = capa.rules.cache.get_cache_path(cache_dir, id)
-
-    assert path.exists()
-    logger.info(f"cached to: {path}")
```

Comment:
is this missing now?

Reply:
The build CI error was caused becasue I didn't `mkdir` the cache directory. When I tested this locally, I already had cache dir created, so it didn't raise an error for me as opposed to in the CI 😄

Regarding omitting these lines of code, I initially removed them because I think they are duplicate as part of [`cache_ruleset`](https://github.com/mandiant/capa/blob/2c1d5592ca714c1bbe54371b1ecfaffc5d09e563/capa/rules/cache.py#L113L132). We end up loading ruleset content, computing cache id, and getting the cache path _twice_.